ui/src/adapters/api/credentials.ts

```diff
@@ -447,43 +532,54 @@ export async function createAuthQRCode({
 }
 
 type IssuedQRCodeInput = {
-  qrCodeLink: string;
   schemaType: string;
+  universalLink: string;
 };
 
 const issuedQRCodeParser = getStrictParser<IssuedQRCodeInput, IssuedQRCode>()(
   z
     .object({
-      qrCodeLink: z.string(),
       schemaType: z.string(),
+      universalLink: z.string(),
     })
-    .transform(({ qrCodeLink, schemaType }) => ({ qrCode: qrCodeLink, schemaType: schemaType }))
+    .transform(({ schemaType, universalLink }) => ({
+      qrCode: universalLink,
```

Comment:
Instead of doing this transformation, we should rename our domain entity so it also names this `universalLink`

Reply:
Now we can remove the type `IssuedQRCodeInput`